### PR TITLE
XW-1282 add test case for checking curated main pages subpages

### PR DESCRIPTION
**Pull request by bognix**
@Wikia/x-wing 
@drets

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsTrackingPixels.java [hunk 1]
```diff
@@ -72,6 +72,22 @@ public void adsTrackingPixelNotSent(String wiki, String[] pixelUrls) {
...
+  @Test(
+      groups = "AdsTrackingPixelsCuratedMainPage",
+      dataProviderClass = AdsDataProvider.class,
+      dataProvider = "adsTrackingPixelsSentCuratedMainPages"
+  )
+  public void adsTrackingPixelSentCuratedMainPages(String wiki, String page, String[] pixelUrls) {
+    networkTrafficInterceptor.startIntercepting();
+
+    // /main/ URLs are not in /wiki/ directory thus we can't use getUrlForPath method and simple concatenation is enough
+    String testedPage = urlBuilder.getUrlForWiki(wiki) + page;
```
Comment (drets) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsTrackingPixels.java]: I would add `getUrlForCuratedMainPage` method to UrlBuilder or adapt `getUrlForPath` method to handle `/main/` case. But if we are not going to write more tests for the curated main pages on Mercury - we can leave it like it is now. 😉

**Comment by drets** [comment]
LGTM 👍

**Comment by drets** [comment]
LGTM 👍

**Comment by vforge** [comment]
LGTM 👍

**Pull request merged**